src/shared/utils/mixpanel.ts

```diff
@@ -0,0 +1,32 @@
+import mixpanel, { Dict } from "mixpanel-browser"
+
+const isProduction = process.env.REACT_APP_ENV === "PRODUCTION"
```

Comment:
I have concerns that these variables won`t work.

The Vite allows to use variables with the prefix VITE_{anything}
And to get access to environmental variables you need to use `import.meta.env.{name-of-variable}`

More information about how variables work:
https://vitejs.dev/guide/env-and-mode.html#env-variables-and-modes

Reply:
Yeah, that's my copy-paste didn't work as I expected